helm/prometheus-rules/templates/alerting-rules/prometheus-operator.rules.yml

```diff
@@ -31,7 +31,7 @@ spec:
     - alert: PrometheusOperatorListErrors
       annotations:
         description: Errors while performing List operations in controller {{`{{`}}$labels.controller{{`}}`}} in {{`{{`}}$labels.namespace{{`}}`}} namespace.
-      expr: (sum by (controller,namespace) (rate(prometheus_operator_list_operations_failed_total{app="prometheus-operator",namespace="{{ .Values.namespace  }}"}[10m])) / sum by (controller,namespace) (rate(prometheus_operator_list_operations_total{app="prometheus-operator",namespace="{{ .Values.namespace  }}"}[10m]))) > 0.4
```

Comment:
2 things, first we own this app so we should technically have been notified anyway. I think we added that back in the day from mixins but we can adjust them if they page too much

Reply:
Actually I think I remember adding some similar exclusions because customer-managed operators were failing too much.
But all right, fine with me, let's try.

Reply:
oh maybe you did but still the namespace is not monitoring anymore :D